packages/table-react/src/ExpandableTableRowController.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import cx from "classnames";
+import { ExpandButton } from "@fremtind/jkl-expand-button-react";
+import { TableCell, TableCellProps, useTableContext } from ".";
+
+export interface ExpandableTableRowControllerProps extends TableCellProps {
+    /** Settes automatisk av ExpandableTableRow */
+    isOpen?: boolean;
+    onClick?: () => void;
+}
+
+export const ExpandableTableRowController: React.FC<ExpandableTableRowControllerProps> = ({
+    isOpen,
+    onClick,
+    children,
+    className,
+    "aria-controls": ariaControls,
+    ...rest
+}) => {
+    if (isOpen === undefined || typeof onClick !== "function") {
+        throw new Error("ExpandableTableRowController must have ExpandableTableRow as parent");
+    }
+
+    const { compact, collapseToList } = useTableContext();
+
+    // pick text from data-th if possible, but only if it's a list
+    const showTextFromTh: string | undefined = collapseToList ? (rest as Record<string, string>)["data-th"] : undefined;
```

Comment:
Jeg tenkte ikke nøye nok gjennom denne biten forrige runde. `collapseToList` er propen som styrer at en tabell _kan_ vises som en liste – det er ikke en indikator på _om_ tabellen vises som en liste. Listevisningen skjer bare det dersom viewporten er smal nok, men i eksempelet er det overstyrt av `data-collapse´-attributtet. Om du fjerner det attributtet og øker bredden på eksempelet ser du at teksten rendres selv når tabellen er i "vanlig" form.

Mulig `ExpandButton` trenger en `span` med en klasse rundt sin `children`, sånn at vi kan styre den "utenfra" med CSS litt enklere. Da vil vi kunne bruke `sr-only`-mixinen på bredder fra og med `medium`.

Det er også ikke helt gunstig at vi har `undefined` som fallback her. I VoiceOver blir knappen bare lest opp som `knapp, minimert`. Det hadde vært fint med en bittelitt mer hjelpsom fallback, som "Ekspander rad" eller noe sånt.

Reply:
> Det er også ikke helt gunstig at vi har undefined som fallback her.

Enig. Det er vel gjort slik for å unngå å vise tekst på knappen dersom man ikke sender inn children, annet enn i listemodus på liten skjerm. Jeg tenker kanskje det er bedre å bruke `hideLabel`-propen i `ExpandButton` til å styre dette basert på `children` og `collapseToList`.